kie-soup-dataset/kie-soup-dataset-elasticsearch/pom.xml

```diff
@@ -224,13 +224,14 @@
       <groupId>com.spotify</groupId>
       <artifactId>docker-client</artifactId>
       <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>org.glassfish.hk2.external</groupId>
-          <artifactId>javax.inject</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
```

Comment:
now the exclusion is dropped, why?

Reply:
This exclusion is added in the `<dependencyManagement>` section below in the `kie-soup-parent` POM, so there is no reason to add it in the `<dependency>` section. Note that rather than excluding `javax.inject`, I excluded `jakarta.inject`, which is necessary to fix the build for the new Spring version (the new Spring version introduces different transitive dependencies compared to the older version, which affects the dependency tree and requires different exclusions)